package.json

```diff
@@ -12,7 +12,7 @@
 	"workspaces": [
 		"packages/*",
 		"fixtures/*",
-		"packages/templates"
+		"packages/templates/packages/create-cloudflare"
```

Comment:
Due to `changesets` using workspaces to determine what to check for changes, we have to specify the only deployable package `create-cloudflare` otherwise changesets breaks on `"packages/templates"` reason being all the templates are missing `"name"` & `"version"` in all the `package.json`

Reply:
I think what we actually want is to move the packages/templates/packages/create-cloudflare to packages/create-cloudflare, right?

Reply:
Along with `workers.new`

Reply:
🤔

Reply:
Looks like this needs changed after the move of templates to the root?